src/styles/components/_dropdown.scss

```diff
@@ -18,21 +20,49 @@
   }
 
   .dropdown-menu {
+    .dso-group-label {
+      color: $grijs-40;
+      font-size: (14 / 16 * 1em);
+      font-weight: 400;
+      margin: 0;
+      padding: ($u1 / 2) $dso-dropdown-hor-padding ($u1 / 4);
+      text-transform: uppercase;
+    }
+
+    ul {
+      margin: 0;
+      padding: 0;
+
+      &.dso-seperator {
+        border-top: 1px solid $grijs-10;
+        margin-top: 11px;
```

Comment:
Is dit een magische maat? Of valt die nog ergens uit te herleiden? `11px`

Reply:
Het is de margin die voorheen op de (lege) li zat.